src/Workspaces/MSBuildTest/MSBuildWorkspaceTests.cs

```diff
@@ -895,20 +864,7 @@ public void TestOpenSolution_WithNonExistentSolutionFile_Fails()
             });
         }
 
-        [ConditionalFact(typeof(VisualStudioMSBuildInstalled)), Trait(Traits.Feature, Traits.Features.MSBuildWorkspace)]
-        public void TestOpenSolution_WithInvalidSolutionFile_Fails()
```

Comment:
Moved these tests to separate files since they behave differently. Such as the type of exception thrown.